provider/dynamodb.go

```diff
@@ -431,6 +443,43 @@ func (store *dynamodbOnlineStore) GetImport(id ImportID) (Import, error) {
 	return S3Import{id: id, status: string(output.ImportTableDescription.ImportStatus), errorMessage: errorMessage}, nil
 }
 
+const maxDynamoBatchSize = 25
+
+func (table dynamodbOnlineTable) BatchSet(items []SetItem) error {
+	if len(items) > maxDynamoBatchSize {
+		return fferr.NewInternalErrorf(
+			"Cannot batch write %d items.\nMax: %d\n", len(items), maxDynamoBatchSize)
+	}
+	serialized := make([]map[string]types.AttributeValue, len(items))
+	for i, item := range items {
+		dynamoValue, err := serializers[table.version].Serialize(table.valueType, item.Value)
+		if err != nil {
+			return err
+		}
+		serialized[i] = map[string]types.AttributeValue{
+			table.key.Feature: &types.AttributeValueMemberS{Value: item.Entity},
+			"FeatureValue":    dynamoValue,
+		}
+	}
+	reqs := make([]types.WriteRequest, len(serialized))
+	for i, serItem := range serialized {
+		reqs[i] = types.WriteRequest{PutRequest: &types.PutRequest{Item: serItem}}
+	}
+	batchInput := &dynamodb.BatchWriteItemInput{
+		RequestItems: map[string][]types.WriteRequest{
+			table.key.ToTableName(): reqs,
+		},
+	}
+	if _, err := table.client.BatchWriteItem(context.TODO(), batchInput); err != nil {
+		return err
```

Comment:
Do you mean to use `fferr` here given we'd be returning the client's error instead of ours?